Governance.md

```diff
@@ -190,20 +249,30 @@ Certain specific types of decisions by the PMC must be made by a special voting
 
 #### 5.3.1 Decisions where special voting procedures are mandatory
```

Comment:
Two questions:
- Should there be a defined comment/discussion period?
- Should there be a voting window, ie 7 days or 14 days? (ie so it's clear when an item hasn't passed?)
  - Related: in 5.3.2 Election decisions, seems to me the 7 nomination and voting windows seem to me too short.  Suggest 14 days in each case.

Reply:
As has come up with discussion of this current pull request, I think it's important that section 5.3 specify a minimum discussion and review period prior to voting.

As drafted, "5.1.1 Consensus Voting" specifies a "7 day review period" prior to voting.

I very much think that "5.1.2 Majority Voting" and 5.1.3. "Super-Majority Voting" should also specify review periods, though in these two cases (presumably more serious issues), I'd suggest 14 days.

Related: "5.3.2 Election decisions" should be revised to specify 14 day nomination and voting periods.

(*We might also want to add a provision to allow a 75% vote of of the PMC to shorten the review and/or voting timeframes in emergency cases)

Reply:
Related: 

* 5.1.2 and 5.1.3 should also specify a maximum voting period, so it's clear when a vote has failed.

* 5.1.2 and 5.1.3 might also want a minimum voting period, to avoid a case where a small number of fast voters could effectively pass a motion that the full set of voters would decline (given time to actually vote)